marklogic-data-hub/src/main/resources/hub-internal-config/security/roles/data-hub-operator.json

```diff
@@ -12,7 +12,9 @@
     "data-hub-step-definition-reader",
     "data-hub-load-data-reader",
     "data-hub-match-merge-reader",
-    "tde-view"
+    "tde-view",
+    "data-hub-saved-query-reader",
+    "data-hub-saved-query-writer"
```

Comment:
I believe "data-hub-saved-query-writer" should be assigned to "data-hub-developer"

Reply:
Users able to run flows and see entity instances should be able to save personal queries. So added the write capability to data-hub-operator.
data-hub-developer inherits data-hub-operator role. So didn't any role for data-hub-developer

Reply:
Yeah, normally the reader role would go to operator and writer to developer, but this is different since an operator has to be able to read and write saved queries.